### PR TITLE
Multilang: Instructions to preview alerts

### PR DESCRIPTION
**Pull request by AlexandreSato**

### CODE REVIEW

File: selfdrive/ui/translations/README.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ openpilot provides a few tools to help contributors manage their translations an
...
    scons -j$(nproc) selfdrive/ui && selfdrive/ui/ui
    ```
+5. Trigger offRoad alerts (clear OffRoad params later):
+   ```shell
+   selfdrive/ui/tests/cycle_offroad_alerts.py
+   ```
+6. Trigger onRoad alerts (hardcode with the alerts you need to see):
+   ```shell
+   selfdrive/debug/cycle_alerts.py
+   ```
```
Comment (sshane) [inline selfdrive/ui/translations/README.md:41]: Can you make this a separate section about viewing translations in the UI?

Comment (AlexandreSato) [reply selfdrive/ui/translations/README.md]: image

File: selfdrive/ui/translations/README.md [hunk 2]
```diff
@@ -58,3 +58,15 @@ selfdrive/ui/tests/create_test_translations.sh && selfdrive/ui/tests/test_transl
...
+
+Latin-based languages often use more space to convey the same message. Then you may want to predict alerts to make sure nothing is overlapping:<br>
+OffRoad alerts (clear OffRoad params later):
+```shell
+selfdrive/ui/tests/cycle_offroad_alerts.py
+```
+Onroad alerts (hardcode with the alerts you need to see):
+```shell
+selfdrive/debug/cycle_alerts.py
+```
```
Comment (sshane) [inline selfdrive/ui/translations/README.md:72]: let's put it below `Testing`, above the footer photo

File: selfdrive/ui/translations/README.md [hunk 3]
```diff
@@ -58,3 +58,15 @@ selfdrive/ui/tests/create_test_translations.sh && selfdrive/ui/tests/test_transl
 
 ---
 ![multilanguage_onroad](https://user-images.githubusercontent.com/25857203/178912800-2c798af8-78e3-498e-9e19-35906e0bafff.png)
+
+### Pre-viewing alerts
```
Comment (sshane) [inline selfdrive/ui/translations/README.md:62]:
```suggestion
### Previewing alerts
```

Comment (AlexandreSato) [reply selfdrive/ui/translations/README.md]: looks nice !!

**Pull request merged**